### PR TITLE
Fix #377

### LINKED ISSUE #377: trace/exporter/jaeger_agent: Could not serialize attribute http.url to tag
I have a simple python app running flask. Configured it per documentation to export to Jaeger. Always get the following error:

```
2018-10-31 11:51:19,540 WARNING Could not serialize attribute             http.url:http://0.0.0.0:5001/ to tag
2018-10-31 11:51:19,549 ERROR
```

Switch Jaeger exporter out with Zipkin or Print and the error does not appear. Tried with both GoogleCloudFormatPropagator and TraceContextPropagator with same result.

Quick example of code to repro the issue:

```
from opencensus.trace.ext.flask.flask_middleware import FlaskMiddleware

from opencensus.trace import config_integration
from opencensus.trace.propagation.trace_context_http_header_format import TraceContextPropagator

from opencensus.trace.exporters import jaeger_exporter
from opencensus.trace.exporters import zipkin_exporter
from opencensus.trace.exporters.transports.background_thread import BackgroundThreadTransport

app = Flask(__name__)

config_integration.trace_integrations(['httplib'])
exporter_jaeger = jaeger_exporter.JaegerExporter(service_name="demo", host_name="localhost", port=14268, transport=BackgroundThreadTransport)
exporter_zipkin = zipkin_exporter.ZipkinExporter(service_name="demo", transport=BackgroundThreadTransport)

middleware = FlaskMiddleware(app, exporter=exporter_jaeger) # error
#middleware = FlaskMiddleware(app, exporter=exporter_zipkin) # works
#middleware = FlaskMiddleware(app, propagator=TraceContextPropagator(), exporter=exporter_jaeger) # error
#middleware = FlaskMiddleware(app, propagator=TraceContextPropagator(), exporter=exporter_zipkin) # works
```

### PR DESCRIPTION
**Pull request by flands**
For both the Zipkin and Jaeger exporters the http.url tag was not
properly getting set because both exporters expected a string.

In addition, added default options for Jaeger just like Zipkin.

### CODE REVIEW

File: opencensus/trace/ext/flask/flask_middleware.py [hunk 1]
```diff
@@ -40,6 +40,9 @@
 SAMPLING_RATE = 'SAMPLING_RATE'
 TRANSPORT = 'TRANSPORT'
 SERVICE_NAME = 'SERVICE_NAME'
+JAEGER_EXPORTER_SERVICE_NAME = 'JAEGER_EXPORTER_SERVICE_NAME'
```
Comment (reyang) [inline opencensus/trace/ext/flask/flask_middleware.py]: I have a general feedback here: probably we want to refactor this code by decoupling the exporter logic and framework integration. How about extracting Jaeger/Zipkin/Ocagent specific thing from Django/Flask ext?

Comment (flands) [reply opencensus/trace/ext/flask/flask_middleware.py]: @reyang seems like a great suggestion -- I was just leveraging what already existed. Can we capture this in a separate github issue?

Comment (reyang) [reply opencensus/trace/ext/flask/flask_middleware.py]: @flands #394 has been created to track this separately.

Comment (reyang) [reply opencensus/trace/ext/flask/flask_middleware.py]: @flands the branch is outdated, would you update it? I can help to merge.

Comment (flands) [reply opencensus/trace/ext/flask/flask_middleware.py]: @reyang done!

**Pull request merged**